**[Back to main page](https://yolanda-ht.github.io/BioinformaticsRandomSeed/)**

# Reference Genome
> Last update: 6/8/2021 <br>
> By: Huitian (Yolanda) Diao

### Reference Genome
A reference genome (also known as a reference assembly) is a digital nucleic acid sequence database, assembled by scientists as a representative example of the set of genes in one idealized individual organism of a species. As they are assembled from the sequencing of DNA from a number of individual donors, reference genomes do not accurately represent the set of genes of any single individual organism. Instead a reference provides a haploid mosaic of different DNA sequences from each donor. There are reference genomes for multiple species of viruses, bacteria, fungus, plants, and animals [[1]](#1).

- [The Genome Reference Consortium (GRC)](https://www.ncbi.nlm.nih.gov/grc) was founded in 2007 to improve the reference genome assemblies of human, mouse and zebrafish.
- [ENSEMBL](http://useast.ensembl.org/index.html) is a scientific project at the European Bioinformatics Institute, which was launched in 1999 in response to the imminent completion of the Human Genome Project. Ensembl aims to provide a centralized resource for geneticists, molecular biologists and other researchers studying the genomes of our own species and other vertebrates and model organisms.
- [UCSC Genome Browser](https://genome.ucsc.edu/cgi-bin/hgGateway) is an on-line, and downloadable, genome browser hosted by the University of California, Santa Cruz (UCSC). It is an interactive website offering access to genome sequence data from a variety of vertebrate and invertebrate species and major model organisms, integrated with a large collection of aligned annotations.

### Genome versus Transcriptome
Genome = all DNA
Transciptome = all transcribed RNA

### Difference between Reference Genomes
GENCODE uses the UCSC convention of prefixing chromosome names with "chr", e.g. "chr1" and "chrM", but Ensembl calls these "1" or "MT". At the time of writing (Ensembl 89), a few transcripts differ due to conversion issues. In addition, around 160 PAR genes are duplicated in GENCODE but only once in Ensembl. The differences affect fewer than 1% of the transcripts. Apart from gene annotation itself, the links to external databases differ [[2]](#2). 

### Genomice Orientation
[Genomic orientation](http://www.imgt.org/IMGTindex/genomicOrientation.php)


## References
<a id="1">[1]</a> 
[Wikipedia - Reference Genome](https://en.wikipedia.org/wiki/Reference_genome).
<a id="2">[2]</a> 
[UCSC Genome FAQ](https://genome.ucsc.edu/FAQ/FAQgenes.html#ens).
