**[Back to main page](https://yolanda-ht.github.io/BioinformaticsRandomSeed/)**

# Batch Effect Removal

> Last update: 05/14/2020 <br>
> By: Huitian (Yolanda) Diao

## Packages
1. <2020>[Cluster similarity spectrum integration of single-cell genomics data](https://www.biorxiv.org/content/10.1101/2020.02.27.968560v1.abstract)
  *Publisher: bioRxiv | Group: Barbara Treutlein | Institute: ETH*
2. <2019>[Fast, sensitive and accurate integration of single-cell data with Harmony](https://www.nature.com/articles/s41592-019-0619-0.epdf?shared_access_token=rDg_Rd07lrFXExt_ySj7V9RgN0jAjWel9jnR3ZoTv0NfDJkKCfDV_X9Mq3lweQmKiXEXxhrebQRjJEZdc-xNv6-7ZN1XotlD_mo5TSS4Z4eWn-kUo6mBwA5dEAKlTfR8OT6E10MZY_E-906ajbzvgg%3D%3D)
  *Publisher: NAT METHODS | Group: Soumya Raychaudhuri | Institute: Harvard*
3. <2018>[Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors](https://www.nature.com/articles/nbt.4091)
  *Publisher: NAT BIOTECH | Group: John C. Marioni | Institute: EBI*

## Metrics
kBET, LISI, ASW, ARI, DEG

## Benchmarks
1. <2020>[A benchmark of batch-effect correction methods for single-cell RNA sequencing data](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1850-9)


