# Regulon analysis
> Last update: 04/29/2020 <br>
> By: Huitian (Yolanda) Diao

1. <2020> [Gene regulatory network reconstruction using single-cell RNA sequencing of barcoded genotypes in diverse environments](https://elifesciences.org/articles/51254) <br>
  *Publisher: eLife | Group: David Gresham | Institute: NYU*

2. <2020> [SimiC: a single cell gene regulatory network inference method with similarity constraints](https://www.biorxiv.org/content/10.1101/2020.04.03.023002v1.abstract) <br>
  *Publisher: bioRxiv | Group: Mikel Hernaez | Institute: UIUC*

3. <2020> [Comparative single-cell trajectory network enrichment identifies pseudo-temporal systems biology patterns in hematopoiesis and CD8 T-cell development](https://www.biorxiv.org/content/10.1101/2020.04.02.021295v3.abstract) <br>
  *Publisher: bioRxiv | Group: Jan Baumbach | Institute: University of Southern Denmark*

4. <2020> [Imputing single-cell RNA-seq data by combining graph convolution and autoencoder neural networks](https://www.biorxiv.org/content/10.1101/2020.02.05.935296v1.abstract) <br>
  *Publisher: bioRxiv | Group: Yuedong Yang | Institute: Sun Yet-sen University*
  
5. <2020> [scTenifoldNet: a machine learning workflow for constructing and comparing transcriptome-wide gene regulatory networks from single-cell data](https://www.biorxiv.org/content/10.1101/2020.02.12.931469v1) <br>
  *Publisher: bioRxiv | Group: James J. Cai | Institute: TAMU*

6. <2020> [CellOracle: Dissecting cell identity via network inference and in sillico gene perturbation](https://www.biorxiv.org/content/10.1101/2020.02.17.947416v1.abstract) <br>
  *Publisher: bioRxiv | Group: Samantha A. Morris | Institute: WUSTL*

7. <2020> [Scedar: a scalable python package for single-cell RNA-seq exploratory data analysis](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007794) <br>
  *Publisher: Plos Computational Biology | Group: Deanne M. Taylor | Institute: U Penn*

8. <2020> [scLM: automatic detection of consensus gene clusters across multiple single-cell datasets](https://www.biorxiv.org/content/10.1101/2020.04.22.055822v1.abstract) <br>
  *Publisehr: bioRxiv | Group: Wei Zhang | Institute: Wake Forest School of Medicine*

9. <2020> [Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data](https://www.nature.com/articles/s41592-019-0690-6) <br>
  *Publisher: Nature Methods | Group: T. M. Murali | Institute: Virginia Tech*

10. <2020> [Revealing dynamics of gene expression variability in cell state space](https://www.nature.com/articles/s41592-019-0632-3) <br>
  *Publisehr: Nature Methods | Group: Dominic Grun | Institute: University of Freiburg*

11. <2020> [Towards inferring causal gene regulatory networks from single cell expression measurements](https://www.biorxiv.org/content/10.1101/426981v1) <br>
  *Publisher: bioRxiv | Group: Sreeram Kannan | Institute: UW*

12. <2020> [Inferring causal gene regulatory networks from coupled single-cell expression dynamics using Scribe](https://www.sciencedirect.com/science/article/abs/pii/S2405471220300363)
  *Publisehr: Cell Systems | Group: Sreeram Kannan | Institute: UW*
  


